testcafe/tests/helpers/utils.js

```diff
@@ -186,6 +186,8 @@ export async function extractZip(pathToZip, extractPath) {
     await fs
       .createReadStream(pathToZip)
       .pipe(unzipper.Extract({ path: extractPath }))
+    const delay = ms => new Promise(res => setTimeout(res, ms))
+    await delay(15000)
```

Comment:
It should be possible to detect the end of the unzipping, see https://github.com/ZJONSSON/node-unzipper#parsepromise-syntax-sugar